ifrs17/Import/ImportStorage.ipynb

```diff
@@ -353,8 +356,8 @@
                 "\n    public IEnumerable<RawVariable> GetRawVariables(string dataNode) => RawVariablesByImportIdentity.TryGetValue(dataNode, out var variableCollection) ? variableCollection : Enumerable.Empty<RawVariable>();",
                 "\n    public IEnumerable<IfrsVariable> GetIfrsVariables(string dataNode) => IfrsVariablesByImportIdentity.TryGetValue(dataNode, out var variableCollection) ? variableCollection : Enumerable.Empty<IfrsVariable>();",
                 "\n    ",
-                "\n    public double[] GetValues(ImportIdentity id, Func<RawVariable, bool> whereClause) => GetRawVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Select(v => v?.Values ?? (double[])null).AggregateDoubleArray();",
-                "\n    public double GetValue(ImportIdentity id, Func<IfrsVariable, bool> whereClause, int projection = 0) => GetIfrsVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Select(v => v?.Values ?? (double[])null).AggregateDoubleArray().ElementAtOrDefault(projection);",
+                "\n    public double[] GetValues(ImportIdentity id, Func<RawVariable, bool> whereClause) => GetRawVariables(id.DataNode).Where(v => new AocStep(v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Select(v => v?.Values ?? (double[])null).AggregateDoubleArray();",
```

Comment:
Should we replace the creation of an AocStep with individual comparison of AocType and Novelty